tico/passes/decompose_fake_quantize.py

```diff
@@ -31,6 +31,8 @@
 def get_quant_type(min: int, max: int) -> torch.dtype:
     if min == 0 and max == 255:
         return torch.uint8
+    if min == 0 and max == 15:
+        return torch.uint8
```

Comment:
I guess that `FoldQuantOps` will create a wrong qparam with this change.

`FoldQuantOps` should set qparam.dtype using `infer_dtype` as `RemoveWeightDequantOp` does.